src/amberscript/parser_buffer_test.cc

```diff
@@ -485,7 +483,8 @@ INSTANTIATE_TEST_CASE_P(
 
 struct BufferData {
   const char* name;
-  DataType type;
+  FormatMode type;
+  size_t num_bits;
```

Comment:
Num bits in what?

Hm. This struct is a param to test code in the same file.  so I'm not too picky about this.